### PR TITLE
remove api v4 docs page since its not really supported

### PR DESCRIPTION
**Pull request by Topherhindman**
## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Review by nghi-ly — COMMENTED** [review]
looks good one question tho, @Topherhindman , should there be a redirect? if this page was visible to people, we can redirect to avoid a 404 for better UX.   

i've only done redirects for MD files tho. lemme know if you need help and i can look into it for you!

**Comment by Topherhindman** [comment]
> looks good one question tho, @Topherhindman , should there be a redirect? if this page was visible to people, we can redirect to avoid a 404 for better UX.
> 
> i've only done redirects for MD files tho. lemme know if you need help and i can look into it for you!

@nghi-ly  Just added another commit to add the redirect

File: _redirects [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/dbt-cloud/api-v4 /docs/dbt-cloud-apis/admin-cloud-api 302
```
Comment (nghi-ly) [inline _redirects]:
```suggestion
/dbt-cloud/api-v4 /docs/dbt-cloud-apis/admin-cloud-api 301
```

Comment (nghi-ly) [reply _redirects]: 301 if you want this to be a permanent change

**Review by nghi-ly — COMMENTED** [review]
thanks for adding the redirect, @Topherhindman ! left one comment

**Review by nghi-ly — APPROVED** [review]

**Pull request merged**